backend/migrations/1656673422615-tasks-effort-and-archive.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class tasksEffortAndArchive1656673422615 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(
+      'alter table tasks add column archived boolean not null default true;',
+    );
+    queryRunner.query('ALTER TABLE tasks ADD COLUMN effort smallint;');
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+       queryRunner.query('alter table task drop effort;');
```

Comment:
Please check the indentation here. I noticed that the project has a Prettier config so you can run it or apply it to your IDE to be ran on every save or even in your local repo to be ran on every commit.